config/tomcat.m4

```diff
@@ -0,0 +1,59 @@
+dnl
+dnl tomcat.m4: Locates the Apache Tomcat/6.0.x and its scripts and jars. 
+dnl
+
+# PGAC_CATALINA_HOME
+# ------------------
+AC_DEFUN([PGAC_CATALINA_HOME],
+[
+  AC_MSG_CHECKING([CATALINA_HOME])
+
+  dnl Use $CATALINA_HOME if specifed, otherwise look for java in the likely places.
+  dnl /usr/local/Cellar/tomcat@6/6.0.45/libexec 
+  dnl /usr/lib/bigtop-tomcat
+  if test -x "${CATALINA_HOME}/bin/catalina.sh"; then
+    TOMCAT="${CATALINA_HOME}"
+  elif test -x "/usr/local/Cellar/tomcat@6/6.0.45/libexec/bin/catalina.sh"; then
```

Comment:
Oh. No, the version should be 6.0.x. I will remove this else option since I just found it's too specific to the path.